testgrid/cmd/configurator/prow.go

```diff
@@ -32,6 +33,9 @@ const testgridCreateTestGroupAnnotation = "testgrid-create-test-group"
 const testgridDashboardsAnnotation = "testgrid-dashboards"
 const testgridTabNameAnnotation = "testgrid-tab-name"
 const testgridEmailAnnotation = "testgrid-alert-email"
+const testgridNumColumnsRecentAnnotation = "testgrid-num-columns-recent"
```

Comment:
README update?

Reply:
To https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#prow-job-configuration now

Reply:
Did it while you were reviewing!

Reply:
Nice XD